src/type/__tests__/enumType.js

```diff
@@ -59,7 +66,31 @@ describe('Type System: Enum Values', () => {
     }
   });
 
-  var schema = new GraphQLSchema({ query: QueryType });
+  var MutationType = new GraphQLObjectType({
+    name: 'Mutation',
+    fields: {
+      favoriteEnum: {
+        type: FavoriteType,
+        args: { color: { type: ColorType } },
+        resolve(value, color) { return color; }
```

Comment:
This test is currently working because the `color` variable you're returning here has the value `{"color":"GREEN"}`. Since the output type is `FavoriteType` which is an Object with a field "color", it accepts this value, and the default resolver accesses the "color" property of that object and returns "GREEN".

However, I believe this is a little more verbose than necessary for the case you're testing.

I think you the "FavoriteType" Object type is unnecessary for testing that mutations will properly coerce variables. I would suggest not returning the entire arguments object, but only the color key, so the output of resolve is instead `"GREEN"`, and then change the type of this field to "ColorType".